### PR TITLE
[Bug fix] Fix bugs for paddle examples

### PR DESCRIPTION
**Pull request by HydrogenSulfate**
1. Fix several paddle code bugs in example.
2. Correct `PDEResidualResampler` to `PDEPointResampler` in all docs and python file(from #764).

### CODE REVIEW

File: examples/pinn_inverse/fractional_Poisson_1d_inverse.py [hunk 1]
```diff
@@ -9,7 +9,14 @@
 
 
 alpha0 = 1.8
+# Backend tensorflow.compat.v1
 alpha = tf.Variable(1.5)
```
Comment (lululxvi) [inline examples/pinn_inverse/fractional_Poisson_1d_inverse.py:13]: Use `dde.Variable`

File: examples/pinn_inverse/fractional_Poisson_2d_inverse.py [hunk 2]
```diff
@@ -9,7 +9,14 @@
 
 
 alpha0 = 1.8
+# Backend tensorflow.compat.v1
 alpha = tf.Variable(1.5)
```
Comment (lululxvi) [inline examples/pinn_inverse/fractional_Poisson_2d_inverse.py:13]: Use `dde.Variable`

File: examples/pinn_inverse/fractional_Poisson_1d_inverse.py (hunk 1 above)
Comment (HydrogenSulfate) [reply examples/pinn_inverse/fractional_Poisson_1d_inverse.py]: Done

File: examples/pinn_inverse/fractional_Poisson_2d_inverse.py (hunk 2 above)
Comment (HydrogenSulfate) [reply examples/pinn_inverse/fractional_Poisson_2d_inverse.py]: Done

File: examples/pinn_inverse/fractional_Poisson_2d_inverse.py [hunk 3]
```diff
@@ -9,7 +9,8 @@
 
 
 alpha0 = 1.8
-alpha = tf.Variable(1.5)
+# Backend tensorflow.compat.v1
```
Comment (lululxvi) [inline examples/pinn_inverse/fractional_Poisson_2d_inverse.py:12]: Delete this comment

Comment (HydrogenSulfate) [reply examples/pinn_inverse/fractional_Poisson_2d_inverse.py]: Done

**Pull request merged**